docs/architecture/grpc-for-wcf-developers/index.md

```diff
@@ -32,36 +32,34 @@ The Docker whale logo is a registered trademark of Docker, Inc. Used by permissi
 
 ## Who should use this guide
 
-This guide was written for developers working in .NET Framework or .NET Core who have previously used WCF and who are seeking to migrate their applications to a modern RPC environment for .NET Core 3.0 and later versions. The guide may also be of use more generally for developers upgrading or considering upgrading to .NET Core 3.0 who want to use the built-in gRPC tools.
+This guide was written for developers working in .NET Framework or .NET Core who have previously used WCF, and who are seeking to migrate their applications to a modern RPC environment for .NET Core 3.0 and later versions. More generally, if you are upgrading, or considering upgrading, to .NET Core 3.0, and you want to use the built-in gRPC tools, this guide is also useful.
 
 ## How you can use this guide
 
-This is a short introduction to building gRPC Services in ASP.NET Core 3.0 with particular reference to WCF as an analogous platform. It explains the principles of gRPC, relating each concept to the equivalent features of WCF, and offers guidance for migrating an existing WCF application to gRPC. It is also useful for developers who have experience of WCF and are looking to learn gRPC to build new services. The sample applications can be used as a template or reference for your own projects, and you are free to copy and reuse code from the book or its samples.
+This is a short introduction to building gRPC Services in ASP.NET Core 3.0, with particular reference to WCF as an analogous platform. It explains the principles of gRPC, relating each concept to the equivalent features of WCF, and offers guidance for migrating an existing WCF application to gRPC. It's also useful for developers who have experience with WCF and are looking to learn gRPC to build new services. You can use the sample applications as a template or reference for your own projects, and you are free to copy and reuse code from the book or its samples.
 
-Feel free to forward this guide to your team to help ensure a common understanding of these considerations and opportunities. Having everybody working from a common set of terminology and underlying principles helps ensure consistent application of architectural patterns and practices.
+Feel free to forward this guide to your team to help ensure a common understanding of these considerations and opportunities. Having everybody working from a common set of terms and underlying principles helps ensure consistent application of architectural patterns and practices.
 
 ## References
 
-- **gRPC web site**  
-  <https://grpc.io>
-- **Choosing between .NET Core and .NET Framework for server apps**  
-  <https://docs.microsoft.com/dotnet/standard/choosing-core-framework-server>
+- [gRPC website](https://grpc.io)
```

Comment:
Keep the previous link format since all the e-books have that link style, which I know are not common but they're supposed to look like this for printing